### PR TITLE
feat: add new file for new bytesize

### PR DESCRIPTION
**Pull request by MaxUlysse**

### CODE REVIEW

File: markdown/events/2021/bytesize-8-nf-core-offline.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Join us for an episode of our **weekly series** of short talks: **“nf-core/bytesize”**.
+
+Just **15 minutes** + questions, we will be focussing on topics about using and developing nf-core pipelines.
+These will be recorded and made available at <https://nf-co.re>
+It is our hope that these talks / videos will build an archive of training material that can complement our documentation.
+Got an idea for a talk? Let us know on the [`#bytesize`](https://nfcore.slack.com/channels/bytesize) Slack channel!
+
+## Bytesize 8: Running pipeline offline
+
+This week, Maxime Garcia ([@maxulysse](http://github.com/maxulysse/)) will present: _**Running pipeline offline.**_
```
Comment (mashehu) [inline markdown/events/2021/bytesize-8-nf-core-offline.md:25]:
```suggestion
This week, Maxime Garcia ([@maxulysse](http://github.com/maxulysse/)) will present: _**Running pipelines offline.**_
```

File: markdown/events/2021/bytesize-8-nf-core-offline.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: "Bytesize 8: Running pipeline offline"
```
Comment (mashehu) [inline markdown/events/2021/bytesize-8-nf-core-offline.md:2]:
```suggestion
title: "Bytesize 8: Running pipelines offline"
```

**Review by mashehu — APPROVED** [review]

File: markdown/events/2021/bytesize-8-nf-core-offline.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+subtitle: Maxime Garcia - SciLifeLab / Karolinska Institutet, Sweden
+type: talk
+start_date: "2021-04-13"
+start_time: "13:00 CET"
+end_date: "2021-04-13"
+end_time: "13:30 CET"
+youtube_embed: https://youtu.be/N1rRr4J0Lps
+location_url:
+    - https://youtu.be/N1rRr4J0Lps
```
Comment (ggabernet) [inline markdown/events/2021/bytesize-8-nf-core-offline.md:11]:
```suggestion
    - https://youtu.be/N1rRr4J0Lps
    - https://zoom.us/j/95310380847
```

**Review by ggabernet — APPROVED** [review]
Just a suggestion to also add the Zoom link under location, the rest LGTM!

**Pull request merged**